### PR TITLE
Add right component with button to go to project edit and options menu with delete option to project view

### PR DESCRIPTION
**Pull request by nickgraz**

### CODE REVIEW

**Comment by nickgraz** [comment]
Current dependencies on/for this PR:
* main
  * **PR #1851** <a href="https://app.graphite.dev/github/pr/terraware/terraware-web/1851?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
    * **PR #1852** <a href="https://app.graphite.dev/github/pr/terraware/terraware-web/1852?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
      * **PR #1853** <a href="https://app.graphite.dev/github/pr/terraware/terraware-web/1853?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
        * **PR #1854** <a href="https://app.graphite.dev/github/pr/terraware/terraware-web/1854?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
          * **PR #1855** <a href="https://app.graphite.dev/github/pr/terraware/terraware-web/1855?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
            * **PR #1856** <a href="https://app.graphite.dev/github/pr/terraware/terraware-web/1856?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
              * **PR #1857** <a href="https://app.graphite.dev/github/pr/terraware/terraware-web/1857?utm_source=stack-comment-icon" target="_blank">Graphite</a>  👈

This [stack of pull requests](https://stacking.dev/?utm_source=stack-comment) is managed by [Graphite](https://app.graphite.dev/github/pr/terraware/terraware-web/1857?utm_source=stack-comment).

**Comment by nickgraz** [comment]
Current dependencies on/for this PR:
* main
  * **PR #1851** <a href="https://app.graphite.dev/github/pr/terraware/terraware-web/1851?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
    * **PR #1852** <a href="https://app.graphite.dev/github/pr/terraware/terraware-web/1852?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
      * **PR #1853** <a href="https://app.graphite.dev/github/pr/terraware/terraware-web/1853?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
        * **PR #1854** <a href="https://app.graphite.dev/github/pr/terraware/terraware-web/1854?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
          * **PR #1855** <a href="https://app.graphite.dev/github/pr/terraware/terraware-web/1855?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
            * **PR #1856** <a href="https://app.graphite.dev/github/pr/terraware/terraware-web/1856?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
              * **PR #1857** <a href="https://app.graphite.dev/github/pr/terraware/terraware-web/1857?utm_source=stack-comment-icon" target="_blank">Graphite</a>  👈

This [stack of pull requests](https://stacking.dev/?utm_source=stack-comment) is managed by [Graphite](https://app.graphite.dev/github/pr/terraware/terraware-web/1857?utm_source=stack-comment).

**Review by karthikbtf — APPROVED** [review]

File: src/components/ProjectView/index.tsx [hunk 1]
```diff
@@ -47,8 +56,37 @@ export default function ProjectView(): JSX.Element {
     </Typography>
   );
 
+  const onOptionItemClick = useCallback((optionItem: DropdownItem) => {
+    switch (optionItem.value) {
+      case 'delete': {
+        // TODO open up delete confirmation
```
Comment (karthikbtf) [inline src/components/ProjectView/index.tsx:62]: is there another jira to delete a project?

**Comment by nickgraz** [comment]
> 
> 
> is there another jira to delete a project?

Yea, it is here https://terraformation.atlassian.net/browse/SW-4491

**Pull request merged**